src/modules/kubernetes.rs

```diff
@@ -0,0 +1,54 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+fn ex(cmd: &str, args: &[&str]) -> Result<String, String> {
```

Comment:
This is a nice helper--so nice, in fact, that I don't think it should stay limited to this scope. This seems like a pretty common operation 

I won't ask you to refactor the entire codebase, but we should probably look into moving it out of this at some point in the future.

(roping in @matchai for his opinion)